src/@orb.yml

```diff
@@ -50,7 +55,8 @@ jobs:
     docker:
       - image: gitguardian/ggshield:<<parameters.tag>>
     environment:
-      CIRCLE_RANGE: <<parameters.base_revision>>...<<parameters.revision>>
+      NUMBER_COMMITS_GH: <<parameters.number_commits_gh>>
+      NUMBER_COMMITS_GL: <<parameters.number_commits_gl>>
```

Comment:
nitpick: `NUMBER_COMMITS_<SERVICE>` does not sound very English. I would prefer `<SERVICE>_COMMIT_COUNT` or `CIRCLECI_<SERVICE>_COMMIT_COUNT`.